routes/brandRouter.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const brandController = require("../controller/brandController");
+
+const router = express.Router();
+
+router.get("/brand", brandController.getAllBrands);
+router.get("/brand/:brandId", brandController.getBrand);
+router.post("/brand", brandController.createBrand);
+router.delete("/brand/:brandId", brandController.deleteBrand);
+router.put("/brand/:brandId", brandController.updateBrand);
```

Comment:
```suggestion
router.get("/", brandController.getAllBrands);
router.get("/:brandId", brandController.getBrand);
router.post("/", brandController.createBrand);
router.delete("/:brandId", brandController.deleteBrand);
router.put("/:brandId", brandController.updateBrand);
```
- 더 상위 라우터인 `index.js`에서 앞에 /brands라고 명시를 해두었기 때문에 하위 라우터에는 brand를 반복해서 쓰실 필요는 없을 것 같습니다